app/tests/integration_test.rs

```diff
@@ -0,0 +1,30 @@
+use assert_cmd::Command;
+
+/// The name of the CLI binary
+const BINARY: &str = "pol";
```

Comment:
Consider `env!("CARGO_BIN_EXE_pol")` instead.

Reply:
Done in [d48c6e8](https://github.com/polarity-lang/polarity/pull/215/commits/d48c6e82cec14457af864a247bea8a506f010de8)